docs/site/java.md

```diff
@@ -0,0 +1,35 @@
+---
+title: JAVA (+ Virgil)
+description: Clojure and Java projects can be developed REPL driven thanks to Virgil
+---
+
+# Mixed Clojure / Java projects
+
+## Mixed builds
+Combined Java and Clojure projects where possible from the first day of Clojure.
+The official Clojure documentation describes 
+[here](https://clojure.org/guides/tools_build#_mixed_java_clojure_build) such a 
+setup incl. a `build.clj`
+
+For Leiningen we have a similar setup documented [here](https://github.com/technomancy/leiningen/blob/github/doc/MIXED_PROJECTS.md) .
+
+## Java development in VSCode
+VSCode has as well [first class support](https://code.visualstudio.com/docs/languages/java) for Java.
```

Comment:
```suggestion
VSCode has as good [first class support](https://code.visualstudio.com/docs/languages/java) for Java.
```

Not sure that's what you intend, but it makes more sense to me when reading. 😄

Reply:
changed